src/App.tsx

```diff
@@ -1,9 +1,34 @@
-function App() {
+import PanelsWrapper from './components/PanelsWrapper';
+import { useEffect, useState } from 'react';
+import { useConfig} from '@/contexts/ConfigContext'
+
+
+
+function App({customConfig}) {
+
+  const [loading, setLoading] = useState(true)
+  const { config, updateConfig} = useConfig()
```

Comment:
updateConfig not exported, code style

Reply:
I think updateConfig is not necessary, since setConfig itself in the ConfigContext can receive a new Config as an argument and update the config. Therefore, I removed updateConfig. This should still be fine, or ?